specs-go/config.go

```diff
@@ -568,3 +570,13 @@ type LinuxIntelRdt struct {
 	// Format: "L3:<cache_id0>=<cbm0>;<cache_id1>=<cbm1>;..."
 	L3CacheSchema string `json:"l3CacheSchema,omitempty"`
 }
+
+// VTPM is used to hold the configuration state of a VTPM
+type VTPM struct {
+	// The directory where the TPM emulator writes the TPM state to
+	StatePath string `json:"statePath"`
+	// Whether to create a certificate for the VTPM
+	CreateCerts bool `json:"createCerts"`
+	// Version of the TPM
+	VTPMversion string `json:"vtpmVersion"`
```

Comment:
This (and `CreatCerts`) should be `omitempty` because they are optional.

Reply:
fixed